flink-connectors/flink-connector-hbase/src/main/java/org/apache/flink/connector/hbase/HBaseEvent.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.connector.hbase;
+
+import org.apache.flink.connector.hbase.sink.HBaseSinkSerializer;
+import org.apache.flink.connector.hbase.source.HBaseSource;
+import org.apache.flink.connector.hbase.source.reader.HBaseSourceEvent;
+
+import org.apache.hadoop.hbase.Cell;
+
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+
+/**
+ * The base HBaseEvent which needs to be created in the {@link HBaseSinkSerializer} to write data to
+ * HBase. The subclass {@link HBaseSourceEvent} contains additional information and is used by the
+ * {@link HBaseSource} to represent an incoming event from HBase.
+ */
+public class HBaseEvent {
+
+    public static final Charset DEFAULT_CHARSET = StandardCharsets.UTF_8;
+
+    private final String rowId;
+    private final String cf;
```

Comment:
`cf` -> `columnFamily`.

Reply:
✅ in 5ba3ba8f10afffc296440e31b51eda0bc411ec22